docs/stencil-docs/customizing-checkout/optimized-one-page-checkout.md

```diff
@@ -2,19 +2,19 @@
 
 <div class="otp" id="no-index">
 
-### On This Page
-- [Working with the Optimized Checkout SCSS File](#working-with-the-optimized-checkout-scss-file)
-- [Configuring the Desktop Viewport](#configuring-the-desktop-viewport)
-- [Classes Available for Customization](#classes-available-for-customization)
-- [Providing Customizable Options in Store Design](#providing-customizable-options-in-store-design)
-- [Currency Conversion Options](#currency-conversion-options)
+### On this page
+- [Working with the optimized checkout SCSS file](#working-with-the-optimized-checkout-scss-file)
+- [Configuring the desktop viewport](#configuring-the-desktop-viewport)
+- [Classes available for customization](#classes-available-for-customization)
+- [Providing customizable options in Store Design](#providing-customizable-options-in-store-design)
+- [Currency conversion options](#currency-conversion-options)
 - [Resources](#resources)
 
 </div> 
 
-## Working with the Optimized Checkout SCSS File
+## Working with the optimized checkout SCSS file
 
-Cornerstone includes an SCSS file that styles the Optimized Checkout page. In your local installation, this file is located at <span class="fn">cornerstone/assets/scss/optimized-checkout.scss</span>.
+Cornerstone includes a SCSS file that styles the Optimized Checkout page. In your local installation, this file is located at <span class="fn">cornerstone/assets/scss/optimized-checkout.scss</span>.

```

Comment:
Should Optimized Checkout be lowercase to match the header?

Reply:
yes it should be lowercase, isn't it showing that way?